semantic/src/site/globals/site.variables

```diff
@@ -85,9 +87,28 @@
 .secondary {
   font-size: 13px;
 }
-.small {
+
+.xx-samll {
   font-size: 12px;
 }
+.x-small {
+  font-size: 13px;
+}
+.small {
+  font-size: 14px;
+}
+.medium {
+  font-size: 15px;
+}
+.large {
+  font-size: 20px;
+}
+.x-large {
+  font-size: 27px;
+}
+.xx-large {
+  font-size: 36px;
+}
```

Comment:
Why classes not @variables?

Reply:
To extend how they are used (on sematic-ui elements). Do you think this wont work?

Reply:
Are you saying that they're over-riding existing Semantic classes? That's cool, but then they should probably be in .overrides, not .variables.

Also -- I'm not sure you can use them the way you have, just putting them inside other definitions. I dunno, maybe its a LESS feature, but it's not valid CSS.